pallets/pallet-web3-names/src/lib.rs

```diff
@@ -496,6 +502,32 @@ pub mod pallet {
 		fn unban_name(name: &Web3NameOf<T>) {
 			Banned::<T>::remove(name);
 		}
+
+		#[cfg(any(feature = "try-runtime", test))]
+		pub fn do_try_state() -> Result<(), &'static str> {
+			// check if for each owner there is a name stored.
+			Owner::<T>::iter().try_for_each(
+				|(w3n, ownership): (Web3NameOf<T>, Web3OwnershipOf<T>)| -> Result<(), &'static str> {
+					ensure!(Names::<T>::contains_key(ownership.owner.clone()), "W3n unknown");
```

Comment:
This check seems redundant, given the line below, no?

Reply:
Well, `Names` and `Owner` should have a 1to1 relation. It can happen that a name record is pointing to an owner, which does not exist. The `check_claiming_preconditions` ensures, that the relation is correct, but I think testing it does not hurt.

Reply:
Owner and names should have a 1 to 1 relation. It might happen, that an owner is pointing to a name which does not exist. Iterating over only one of these storage, would not reveal these issues.